### PR TITLE
Added CocosCreator version 1.2.0

### PR DESCRIPTION
**Pull request by cybertk**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/cocoscreator.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask 'cocoscreator' do
+  version '1.2.0'
+  sha256 '98bba2f8f78998c0c68917a8ad56d7f4105c36c0f2c150268a108b8e4f2f9388'
+
+  # appget.cn was verified as official when first introduced to the cask
+  url 'http://cocostudio.download.appget.cn/CocosCreator/v1.2.0/CocosCreator_v1.2.0_2016080301.dmg'
```
Comment (fanquake) [inline Casks/cocoscreator.rb]: You can use #{version} in the url stanza

File: Casks/cocoscreator.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  name 'CocosCreator'
+  homepage 'http://www.cocos2d-x.org/'
+  license :mit
+
+  app 'CocosCreator.app'
+  binary "#{appdir}/CocosCreator.app/Contents/Resources/cocos2d-x/tools/cocos2d-console/bin/cocos", target: 'cocos'
```
Comment (vitorgalvao) [inline Casks/cocoscreator.rb]: Remove `, target: 'cocos'`. Unecessary.

File: Casks/cocoscreator.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/cocoscreator.rb]: Link I get is `http://cdn.cocos2d-x.org/CocosCreator_v1.2.0_2016080301.dmg`. Use that one instead: `url "http://cdn.cocos2d-x.org/CocosCreator_v#{version}.dmg"`.

File: Casks/cocoscreator.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+cask 'cocoscreator' do
+  version '1.2.0'
```
Comment (vitorgalvao) [inline Casks/cocoscreator.rb]: `1.2.0_2016080301`.

File: Casks/cocoscreator.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  sha256 '98bba2f8f78998c0c68917a8ad56d7f4105c36c0f2c150268a108b8e4f2f9388'
+
+  # appget.cn was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/cocoscreator.rb]: Remove comment (after updating `url`).

**Comment by vitorgalvao** [comment]
> - [ ] `brew cask install {{cask_file}}` worked successfully.

Why is it unchecked? How did it fail?

**Comment by jawshooah** [comment]
Thank you for the contribution. It needed some fixes, so they were made in ffb8c463098ab44bc7c835735c185c2faff334d9. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**